main.py

```diff
@@ -1,10 +1,53 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for player_name, player_data in players.items():
+        player_guild = player_data.get("guild")
+        player_race_name = player_data.get("race").get("name")
```

Comment:
It can return Exception None Type. Use:
```suggestion
        player_race_name = player_data.get("race", {}).get("name")
```
Fix in other places